### PR TITLE
Maint-26893: show alert message when adding group

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
**Problem:** none popup shown when user enter invalid character for group name.
**Solution:** add regex to check group name

### CODE REVIEW

File: webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue [hunk 1]
```diff
@@ -161,6 +170,13 @@ export default {
       this.drawer = true;
     },
     saveGroup(event) {
+      const regex =  /^[a-zA-Z0-9-_]+$/;
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue:164]: It is better if you handle this at Rest service level as it is already done for validating if name or label are empty.

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue (hunk 1 above)
Comment (hassenelkamel-exo) [reply webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue:164]: done

File: webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue [hunk 2]
```diff
@@ -161,6 +161,15 @@ export default {
...
+      const isValid = regex.test(this.group.groupName);
+      if (!isValid){
+        this.error = this.$t('GroupsManagement.popup.message');
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue]: you can call `this.handleError(this.$t('GroupsManagement.popup.message'))` instead.
+
`GroupsManagement.popup.message` is not a significant key, you can use `GroupsManagement.error.invalidGroupName` for example.

File: webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue [hunk 3]
```diff
@@ -96,7 +96,7 @@ export default {
     saving: false,
     confirmNewPassword: null,
     group: {},
-    parentGroup: null,
+    parentGroup: null
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue:99]: Just for curiosity, why this change ?

Comment (hassenelkamel-exo) [reply webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue:99]: in the first time, i add a popup and new variable popupBodyMessage and ali tell me that you don't need to add those so I revert code

File: webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue [hunk 4]
```diff
@@ -186,9 +195,8 @@ export default {
       }).then(resp => {
         if (!resp || !resp.ok) {
           if (resp.status === 400) {
-            return resp.text().then(error => {
-              this.fieldError = error;
-              throw new Error(error);
+            return resp.text().then(() => {
+              throw new Error(this.$t('GroupsManagement.popup.message'));
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue]: This change will affect the other errors to not display properly

**Review by boubaker — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue (hunk 4 above)
Comment (hassenelkamel-exo) [reply webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue]: ok done

File: webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue (hunk 2 above)
Comment (hassenelkamel-exo) [reply webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue]: done

**Review by boubaker — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]
LGTM
